docs/UsersGuide/source/BackgroundInfo/Components.rst

```diff
@@ -59,20 +61,18 @@ The Model Evaluation Tools (MET) package is a set of statistical verification to
+METplus comes preinstalled with :term:`spack-stack` but can also be installed on other systems individually or as part of :term:`HPC-Stack` installation. Users on systems without a previous installation of METplus can follow the :ref:`MET Installation Guide <met:installation>` and :ref:`METplus Installation Guide <metplus:install>` for individual installation. Currently, METplus *installation* is only supported as part of spack-stack installation; users attempting to install METplus individually or as part of HPC-Stack will need to direct assistance requests to the METplus team. However, METplus *use* is supported on any system with a functioning METplus installation.
 
-The core components of the METplus framework include the statistical driver, MET, the associated database and display systems known as METviewer and METexpress, and a suite of Python wrappers to provide low-level automation and examples, also called use cases. MET is a set of verification tools developed for use by the :term:`NWP` community. It matches up grids with either gridded analyses or point observations and applies configurable methods to compute statistics and diagnostics. Extensive documentation is available in the `METplus User's Guide <https://metplus.readthedocs.io/en/main_v4.1/Users_Guide/index.html>`__ and `MET User's Guide <https://met.readthedocs.io/en/main_v10.1/index.html>`__. Documentation for all other components of the framework can be found at the Documentation link for each component on the METplus `downloads <https://dtcenter.org/community-code/metplus/download>`__ page.
+The core components of the METplus framework include the statistical driver, MET, the associated database and display systems known as METviewer and METexpress, and a suite of Python wrappers to provide low-level automation and examples, also called use cases. MET is a set of verification tools developed for use by the :term:`NWP` community. It matches up grids with either gridded analyses or point observations and applies configurable methods to compute statistics and diagnostics. Extensive documentation is available in the :doc:`METplus User's Guide <metplus:index>` and :doc:`MET User's Guide <met:index>`. Documentation for all other components of the framework can be found at the *Documentation* link for each component on the METplus `downloads <https://dtcenter.org/community-code/metplus/download>`__ page.
```

Comment:
I didn't know that METviewer is installed as part of the METplus installation.  I wonder if it can be called in batch mode.  Just musing to myself...

Reply:
@ulmononian Can you speak to this? 
@JeffBeck-NOAA @gsketefian FWIW, this is just the info that was passed down to me--it's been in past releases, too, but if it's wrong, better to correct it asap!

Reply:
@JeffBeck-NOAA I think "the statistical driver, MET," is trying to say "the statistical driver known as MET".

Reply:
Just a comment:  I have never used METexpress.  After reading the description of it [here](https://dtcenter.org/community-code/metexpress), I can't really say how it differs from METViewer or why two systems are necessary.  @JeffBeck-NOAA , is METexpress the GSL version of METviewer (and thus is not really available to the public)?

Reply:
"It matches up grids" --> "It matches up gridded forecast fields"

Reply:
Corrected wording based on @gsketefian 's suggestions. Also checked the [METplus website](https://dtcenter.org/community-code/metplus) and [documentation](https://metplus.readthedocs.io/en/latest/), where it lists the "core components of the [METplus] framework," which supposedly include METviewer and METexpress.